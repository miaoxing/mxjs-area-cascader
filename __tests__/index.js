import React from 'react';
import {render} from '@testing-library/react';
import AreaCascader from '..';

describe('test', () => {
  test('basic', () => {
    const result = render(<AreaCascader/>);

    expect(result.container).toMatchSnapshot();
  });
});
