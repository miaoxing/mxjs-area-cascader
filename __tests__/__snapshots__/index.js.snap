// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test basic 1`] = `
<div>
  <div
    class="form-group row"
  >
    <label
      class="text-sm-right  form-label col-form-label col-sm-2"
      for="province"
    >
      <span
        class="text-warning"
      >
        * 
      </span>
      省份
    </label>
    <div
      class="col-sm-4"
    >
      <select
        class="form-control"
        id="province"
        name="province"
        required=""
      >
        <option
          value=""
        >
          请选择
        </option>
      </select>
    </div>
  </div>
  <div
    class="form-group row"
  >
    <label
      class="text-sm-right  form-label col-form-label col-sm-2"
      for="city"
    >
      <span
        class="text-warning"
      >
        * 
      </span>
      城市
    </label>
    <div
      class="col-sm-4"
    >
      <select
        class="form-control"
        id="city"
        name="city"
        required=""
      >
        <option
          value=""
        >
          请选择
        </option>
      </select>
    </div>
  </div>
  <div
    class="form-group row"
  >
    <label
      class="text-sm-right  form-label col-form-label col-sm-2"
      for="area"
    >
      <span
        class="text-warning"
      >
        * 
      </span>
      区域
    </label>
    <div
      class="col-sm-4"
    >
      <select
        class="form-control"
        id="area"
        name="area"
        required=""
      >
        <option
          value=""
        >
          请选择
        </option>
      </select>
    </div>
  </div>
</div>
`;
